test/common/README.md

```diff
@@ -60,6 +60,19 @@ On non-Windows platforms, this always returns `true`.
 
 Creates a 10 MB file of all null characters.
 
+### `debug(message[, ...args])`
```

Comment:
Instead of adding another common functionality that may be replaced in each file with a single line, what about using a lint rule to specify that debuglog in our tests may only contain `'test'` as argument? And the test that actually checks debuglog could just get an exception.

Reply:
Ooh, I like that.

Reply:
so long as we can actually move towards using `debuglog()` consistently... that would end up requiring duplicated boilerplate in every test file...

```js
const debug= require('util').debuglog('test');
```

Perhaps we could also have a lint rule flagging uses of `console.log()` and `console.error()` that are not explicitly excepted?

Reply:
I'd rather a utility function in common than a lint rule that forces people to write boilerplate code, even if its short boilerplate code.